nucypher/blockchain/eth/agents.py

```diff
@@ -1364,16 +1364,21 @@ def is_claiming_available(self) -> bool:
         result = self.contract.functions.isClaimingAvailable().call()
         return result
 
-    @property
-    @contract_api(CONTRACT_ATTRIBUTE)
+    @contract_api(CONTRACT_CALL)
+    def estimate_verifying_correctness(self, gas_limit: int, gas_to_save_state: int = 30000) -> int:  # TODO - #842: Gas Management
+        """Returns how many bidders will be verified using specified gas limit"""
+        return self.contract.functions.verifyBiddingCorrectness(gas_to_save_state).call({'gas': gas_limit})
+
+    @contract_api(CONTRACT_CALL)
     def next_bidder_to_check(self) -> int:
```

Comment:
What's the reason to take away the `@propery` decorator?

Reply:
I asked @KPrasch  about this, and decided to do that, not very strong opinion at least from my side

Reply:
Good change IMO. `estimate_verifying_concreteness` is a verb.

Reply:
@KPrasch You're reading the diff wrong. My comment is about `next_bidder_to_check`, which returns an index. Following that grammatical criteria, this should be a property, hence my question.

Reply:
Oh you're right!

In that case - I reference #2022.